### PR TITLE
Added a testimonial section

### PR DESCRIPTION
**Pull request by CBID2**
## Fixes Issue
[#2682 ](https://github.com/EddieHubCommunity/LinkFree/issues/2682)

## Check List (Check all the applicable boxes) 

- [X ] My code follows the code style of this project.
- [X ] My change requires changes to the documentation.
- [ X] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ X] This PR does not contain plagiarized content.
- [ X] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2683"></a>

### CODE REVIEW

File: data/CBID2.json [hunk 1]
```diff
@@ -35,3 +35,4 @@
     }
   ]
 }
+"testimonials": ["loftwah"],
```
Comment (loftwah) [inline data/CBID2.json:38]: It should look like

```json
 ],
  "testimonials": [
    "loftwah"
  ]
}

```

**Review by CBID2 — COMMENTED** [review]
Made the change

File: data/CBID2.json [hunk 2]
```diff
@@ -35,3 +35,6 @@
     }
   ]
 }
+"testimonials": [
```
Comment (eddiejaoude) [inline data/CBID2.json:38]: This section needs to go inside the `}` between line `36` and `37`

Comment (CBID2) [reply data/CBID2.json:38]: Ok

Comment (CBID2) [reply data/CBID2.json:37]: > This section needs to go inside the `}` between line `36` and `37`

I think I got it

Comment (eddiejaoude) [reply data/CBID2.json:37]: Now you just need a `,` on line `36`

Comment (CBID2) [reply data/CBID2.json]: Done

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Pull request merged**